### PR TITLE
Add feature CLI back

### PR DESCRIPTION
**Pull request by StrongMonkey**

### CODE REVIEW

**Comment by StrongMonkey** [comment]
This PR adds `rio system` as a command to include system logs and system feature as darren want that in UX. Also `rio system feature` can list descriptions of features.

File: cli/main.go [hunk 1]
```diff
@@ -216,16 +216,6 @@ func main() {
 			appName+" stage [OPTIONS] SERVICE NEW_REVISION",
 			""),
 
-		builder.Command(&systemlogs.SystemLogs{},
-			"View system log for Rio management plane",
-			appName+" systemlogs",
-			""),
-
-		builder.Command(&systemconfig.SystemConfig{},
```
Comment (daxmc99) [inline cli/main.go]: Looks like we also drop rio systemlogs & rio systemconfig? Are we intending that `rio features` be the replacement for `rio systemlogs` in some way? Rio `systemlogs` is currently in our bug report template

File: cli/cmd/system/viewconfig.go [hunk 2]
```diff
@@ -0,0 +1,140 @@
+package system
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+
+	"github.com/rancher/rio/cli/pkg/tables"
+
+	"github.com/rancher/rio/cli/pkg/builder"
```
Comment (cbron) [inline cli/cmd/system/viewconfig.go]: nit: space here and below

File: cli/main.go (hunk 1 above)
Comment (cbron) [reply cli/main.go]: Looks like `rio systemlogs` got moved to `rio system logs`, but we will need to update bug report

File: cli/cmd/system/viewconfig.go [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+	"github.com/rancher/rio/cli/pkg/clicontext"
+	"github.com/rancher/rio/pkg/config"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func System(app *cli.App) cli.Command {
+	logs := builder.Command(&Logs{},
+		"View system logs",
+		app.Name+" system logs",
+		"")
```
Comment (cbron) [inline cli/cmd/system/viewconfig.go]: Need to update CLI-ref docs

**Comment by StrongMonkey** [comment]
@daxmc99 @cbron Changes have been pushed.

**Pull request merged**